sormas-api/src/main/java/de/symeda/sormas/api/caze/classification/ClassificationCriteriaDto.java

```diff
@@ -53,6 +54,8 @@
 	@JsonSubTypes.Type(value = ClassificationXOfCriteriaDto.class, name = "ClassificationXOfCriteriaDto"),
 	@JsonSubTypes.Type(value = ClassificationAllSymptomsCriteriaDto.class, name = "ClassificationAllSymptomsCriteriaDto"),
 	@JsonSubTypes.Type(value = ClassificationEventClusterCriteriaDto.class, name = "ClassificationEventClusterCriteriaDto") })
+@AuditedClass
```

Comment:
Should not be audited. Classification rules can't be edited by users and are not secrect